app/assets/stylesheets/_projects-show.scss

```diff
@@ -11,6 +11,19 @@ header {
     margin-top: 10px;
     float: right;
   }
+  
+  .prev-next-links {
+    margin-bottom: 5px;
+    overflow: overlay;
+
+    .prev-project {
+      float: left;
+    }
+
+    .next-project{
+      float: right;
+    }
+  }
```

Comment:
This isn't working in Firefox (latest, OS X). Also, it doesn't appear to be picking up the link styles from the rest of the site (links are rendering blue underlined instead of pink and not underlined).

![screen shot 2018-08-06 at 11 05 36 am](https://user-images.githubusercontent.com/2909/43724808-eb8409e0-9968-11e8-8c4c-cf1d223ec73f.png)

Reply:
Firefox: ok, checking.

Links: you mean, the Next/Prev Project links should be pink, like here for example? https://www.awesomefoundation.org/en/about_us

I'll do, but you sure this is a good idea? We seem to have navigation or utility links gray/black all over the website, and only inline links are pink (inside a page content).